lib/degica_datadog/config.rb

```diff
@@ -57,6 +64,10 @@ def statsd_port
       def tracing_port
         datadog_agent_uri&.port || 8126
       end
+
+      def inspect
+        "DegicaDatadog::Config<enabled?=#{!!enabled?} service=#{service} version=#{version} environment=#{environment} repository_url=#{repository_url} datadog_agent_host=#{datadog_agent_host} statsd_port=#{statsd_port} tracing_port=#{tracing_port}>" # rubocop:disable Layout/LineLength
```

Comment:
Just out of curiosity, where on DD will we be able to see this?

Reply:
This won't be on DD, but it's what you get when you open up an `irb` and check out `DegicaDatadog::Config`. I ended up checking a lot of these values over and over while tweaking settings, so having them all easily accessible might help in the future when we're debugging something like data not coming through or being wrongly tagged.